Assignment/solution_student/index.html

```diff
@@ -1,35 +1,34 @@
-
-Sample Content
-——-
-
-ABOUT ME PAGE
-
-I ENJOY LIFE AS A DEVELOPER
-
-I'm Joe Johnson, a Developer based in NYC. I Have ten years of experience in the graphic design world, specializing in the creation of responsive websites.
-
-
-RESUME PAGE
-
-Experience
-        
-I recently graduated from a Front-End Web Development course at General Assembly where I learned HTML, CSS, JavaScript, jQuery and how to be an awesome Front-End Web Developer! During my spare time, I enjoy senior water aerobics, pickling, and spending time with my goldfish, Jerry.
-    
-Relevant Experience
-    
-Web Developer, Relaxr 2015 - present
-Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices.
-
-Web Developer, Startup Matchmaker 2015 - present
-Used a design team's wireframes to develop this company's responsive homepage.
-
-Web Developer, CitiPix 2015 – present
-Built web app prototype that allows users to store and quickly retrieve photos using keywords.
-
-
-EDUCATION
-
-General Assembly — FEWD 2015
-College of Soft Knocks — B.S. 2009
-School of Hard Knocks — 2005 (Honors)
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<title>joe johnson about me</title>
+<link rel="stylesheet" href="Css/style.css">
+</head>
+
+<body>
+	<div>
+	
+	<h1><em>About me</em> | <a href="resume.html">Portfolio</a></h1>
+	
+	<h2>JOE JOHNSON</h2>
+
+	<div class="body">
+
+	<h3>I ENJOY LIFE AS A DEVELOPER</h3>
+		<img src="avatar.png" ALT="pic of joe" class="avatar">
+	<p>I'm Joe Johnson, a Developer based in NYC. I have ten years <br>of experience in the graphic design world, specializing in the <br> creation of responsive websites.</p>
+
+	</div>
+	<br>
```

Comment:
Avoid using the `<br>` tag to space things out — instead use the `padding` or `margin` properties in CSS